components/dialog/README.md

```diff
@@ -19,8 +19,13 @@ The `d2l-dialog` element is a generic dialog that provides a slot for arbitrary
   <d2l-button slot="footer" dialog-action>Cancel</d2l-button>
 </d2l-dialog>
 ```
+Open the dialog declaratively using a boolean attribute `opened`:
 
-Open the dialog by calling the `open` method to return a promise:
+```html
+<d2l-dialog ?opened="${this.clicked}"></d2l-dialog>
```

Comment:
To avoid any further confusion, I'd probably change the bound property to something like `this.someProp` rather than `this.clicked`.

Reply:
With the binding approach, there is a real risk that the consumer's bound property (`this.clicked`) will get out of sync with the dialog `opened` state. If they don't sync that prop in a `d2l-dialog-close` event handler, the dialog will open once, but not a second time. I think it would be good to add that to this snippet and perhaps move the event info up here.